R/txtProgressBarETA.R

```diff
@@ -97,7 +97,7 @@ txtProgressBarETA <- function (min = 0, max = 1, initial = 0, char = "=", width
   # Kevin - Adjust width based on Windows or *nix platforms
   nw <- nchar(char, "w")
   if (.autoWidth) {
-    if (.Platform$OS.type == "windows" || Sys.getenv("COLUMNS") == "") {
+    if (.Platform$OS.type == "windows" | Sys.getenv("COLUMNS") == "") {
```

Comment:
@kvnkuang I'm curious. Why make this change? Does it have to do with recent changes in R? (I think I remember some news item about it). I'm asking because to me the short-circuit `||` seems more natural to me when I expect both conditions to be of length 1, but I'm wondering if I should change to `|` for some cases.

Reply:
Hi Scott, thanks for reaching out 😄 I changed it for consistency reasons. I think it's safer to use the short form (|) because:
1. When both conditions are length 1, short and long form perform exactly the same
2. When any condition is length > 1, the short form will throw a warning but the long form will just evaluate the first element in that condition.

Reply:
You're welcome! I hope this helps 😃